js/privatebin.js

```diff
@@ -42,6 +42,148 @@ jQuery(document).ready(function() {
     $.PrivateBin.Controller.init();
 });
 
+var globalScrollPosition;
```

Comment:
Would it be possible to move these variables and methods into an existing or a new class in the main PrivateBin namespace instead of "polluting" the global namespace? We use a pattern similar to the one described for example [here](https://dev.to/mrfrontend/javascript-basics-how-to-create-private--public-functions-in-javascript-with-the-module-pattern-3b3o) in order to keep internal state variables and public/private methods separate from each others scopes. It also eases unit testing, if one wants to add such (not required for this PR).